SCClassLibrary/Common/Quarks/Git.sc

```diff
@@ -80,7 +80,11 @@ Git {
 		// all tags
 		// only includes ones that have been fetched from remote
 		^tags ?? {
-			raw = this.git(["for-each-ref --format='%(refname)' --sort=taggerdate refs/tags"]);
+			if(thisProcess.platform.name !== 'windows', {
+				raw = this.git(["for-each-ref --format='%(refname)' --sort=taggerdate refs/tags"]);
```

Comment:
just a note to clarify for others: sh family of shells require quote marks

```
crucial-library ❯ git for-each-ref --format=%(refname) --sort=taggerdate refs/tags
zsh: missing end of string
```

@bagong is saying that windows doesn't want the quote, otherwise he gets:

```
C:\Users\Rainer\AppData\Local\SuperCollider\downloaded-quarks\crucial-library>git for-each-ref --format='%(refname)' --sort=taggerdate refs/tags
…
'refs/tags/for-SC-3.4.1'
'refs/tags/for-sc-3.5rc1'
'refs/tags/4.0.0'
'refs/tags/4.0.1'
'refs/tags/4.1'
'refs/tags/4.1.1'
'refs/tags/docs-syntax-fixes'
'refs/tags/ilisp-help'
'refs/tags/4.1.3'
'refs/tags/4.1.4'
'refs/tags/4.1.5'
```